trtools/compareSTR/compareSTR.py

```diff
@@ -713,12 +736,86 @@ def bad_range():
             common.WARNING("Cannot have range portion of --region "
                            "start-end where end <= start")
             return 1
-    except ValueError: # an int() cast failed
+    except ValueError:  # an int() cast failed
         return bad_range()
 
     return 0
 
 
+def handle_overlaps(args: Any = None) \
```

Comment:
Until we have a defined implementation for any value of args aside from None, I'd rather not have this here, and just have the handler function. Because if we never implement any other uses for handle_overlaps then this just leaves the code in a more confusing state. And if we do implement another use case here, it should be easy to add handle_overlaps() back in.

Reply:
I agree. I've removed this function.